docs/en/agent/tracing-metrics-logging.md

```diff
@@ -0,0 +1,41 @@
+# Tracing, Metrics and Logging with Go Agent
+
+All plugins in SkyWalking Go Agent are designed to provide functionality for distributed tracing, metrics, and logging data. 
+For a detailed list of supported plugins, [please refer to the documentation](./support-plugins.md). 
+This document aims to provide you with some configuration information for your usage. 
+Please ensure that you have followed the [documentation to successfully install the SkyWalking Go Agent into your application](../setup/gobuild.md).
+
+## Tracing
+
+Distributed tracing is the most common form of plugin in the Go Agent, and it becomes active with each new incoming request. By default, all plugins are enabled. For a specific list of plugins, please [refer to the documentation](./support-plugins.md#tracing-plugins).
+
+If you wish to disable a particular plugin to prevent enhancements related to that plugin, please consult the [documentation on how to disable plugins](../advanced-features/plugin-exclusion.md).
+
+The basic configuration is as follows:
+
+| Name                    | Environment Key   | Default Value         | Description                                                                                                                               |
+|-------------------------|-------------------|-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| agent.service_name      | SW_AGENT_NAME     | Your_Application_Name | The name of the service which showed in UI.                                                                                               |
+| agent.instance_env_name | SW_AGENT_INSTANCE |                       | To obtain the environment variable key for the instance name, if it cannot be obtained, an instance name will be automatically generated. |
```

Comment:
I think these two should be global.

Reply:
I have added a registration mechanism section to descript it.